src/file.rs

```diff
@@ -85,11 +86,15 @@ impl FileLoggerBuilder {
         level: Severity,
```

Comment:
I prefer to use a builder or option struct instead of many arguments.
(e.g., Export `KVFilterParameters` and use it as the argument)

Reply:
done